actions/vrfv2_helpers.go

```diff
@@ -2,7 +2,6 @@ package actions
 
```

Comment:
Most of this file just needs changes reverted and should just have your client/networks changes.